main.go

```diff
@@ -2,11 +2,10 @@ package main
 
 import (
 	"github.com/armory/armory-cli/cmd"
-	"github.com/spf13/cobra"
 	"os"
 )
 
 func main() {
 	rootCmd := cmd.NewCmdRoot(os.Stdout, os.Stderr)
-	cobra.CheckErr(rootCmd.Execute())
```

Comment:
This was never necessary since `Execute()` already prints the error unless we added the silence errors configuration to the root command, which we have not.